Dockerfile

```diff
@@ -21,4 +21,8 @@ COPY --from=sbt-plugins-cache /root/.sbt ${JENKINS_HOME}/.sbt
 USER root
 RUN chown -R jenkins ${JENKINS_HOME}
 
+RUN export SDKMAN_DIR="${JENKINS_HOME}/.sdkman" && curl -s "https://get.sdkman.io" | bash
```

Comment:
This is probably fine for now, but pulling sdkman in this way reduces the repeatability of this build. Is there a way to grab a specific version?

Reply:
Cross posting from @aquilesg and I's thread [here](https://github.com/Dwolla/jenkins-agent-docker-nvm-sbt/pull/21#discussion_r1149458247)!